cuda/matrix/batch_dense_kernels.cu

```diff
@@ -60,11 +60,12 @@ namespace cuda {
 namespace batch_dense {
 
 
-constexpr auto default_block_size = 512;
+constexpr auto default_block_size = 256;
```

Comment:
I guess you experimented and found that 256 was a better default block size that 512 ? Do you know why ?

Reply:
That's a good question. I don't remember why I changed this. I think it was while I was optimizing the scale-by-vector operation. In any case it matters little because most of the time is spent in the solver apply kernel.